src/gt4py/gt_src_manager.py

```diff
@@ -22,8 +22,15 @@
 
 
 _DEFAULT_GRIDTOOLS_VERSION = 2
-# TODO: GT2 release with CUDA SID adapter
-_GRIDTOOLS_GIT_BRANCHES = {1: "v1.1.4", 2: "v2.1.0"}
+
+_GRIDTOOLS_GIT_REPO = {
+    1: "https://github.com/GridTools/gridtools.git",
+    2: "https://github.com/GridTools/gridtools.git",
+}  # TODO reset to main repo
+_GRIDTOOLS_GIT_BRANCHES = {
+    1: "v1.1.4",
+    2: "v2.2.0",
+}
```

Comment:
There's nothing in gt4py (capable of) using GT 1 anymore, might as well remove the option from the source manager.  Not necessarily the scope of this PR though.

Reply:
Yes, and thanks to pointing to this line which contains a TODO that I should remove...

Reply:
I agree with @gronerl that we should remove GT1 handling in source manager. If it's not done in this PR then I think it's worth to add a TODO at least.